### PR TITLE
fix(dev-infra): allow build-worker to be used in forked process

### PR DESCRIPTION
**Pull request by josephperrott**
Generates a local copy of the build-worker file to allow it to be loaded
at runtime in a forked process.

### CODE REVIEW

File: dev-infra/BUILD.bazel [hunk 1]
```diff
@@ -42,10 +39,12 @@ rollup_bundle(
     ],
 )
 
-generated_file_test(
-    name = "local_ng_dev",
-    src = "ng-dev.js",
-    generated = "cli_rollup",
+ng_dev_rolled_up_generated_file(
```
Comment (devversion) [inline dev-infra/BUILD.bazel:42]: Should we add a comment here that explains why we need this?

File: dev-infra/index.bzl [hunk 2]
```diff
@@ -3,7 +3,22 @@
 # Use of this source code is governed by an MIT-style license that can be
 # found in the LICENSE file at https://angular.io/license
 
-# File is currently empty but serves as indicator for `rules_nodejs` and instructs it to
-# preserve the content output in the NPM install workspace. This allows consumers to use
-# rules and targets from within Bazel. e.g. by using `@npm//@angular/dev-infra-private/<..>`.
-# See: https://github.com/bazelbuild/rules_nodejs/commit/4f508b1a0be1f5444e9c13b0439e649449792fef.
```
Comment (devversion) [inline dev-infra/index.bzl:9]: Do you mind if we keep the comment somehow (surely adjusting it though). I think it is good Information that we should preserve here.

**Review by devversion — APPROVED** [review]
LGTM.

**Review by gkalpak — APPROVED** [review]
:+1:

**Pull request closed without merging**